integration-tests/utils/integration-utils.js

```diff
@@ -319,6 +319,20 @@ const withScreenshot = _.curry((testName, fn) => async options => {
   }
 })
 
+// Emitted when the page crashes
+const logError = page => {
+  const handle = msg => console.error('page.error', msg)
+  page.on('error', handle)
+  return () => page.off('error', handle)
+}
+
+// Emitted when an uncaught exception happens within the page
+const logPageError = page => {
+  const handle = msg => console.error('page.error', msg)
```

Comment:
Should the messages for logging `error` and `page.error` events be different? Currently, both are `page.error`.

Reply:
👍🏻 Updated for clarity.